package.json

```diff
@@ -1,176 +1,177 @@
-    "lint-staged": "15.1.0",
-    "ng-packagr": "17.0.3",
-    "ngx-cva-test-suite": "2.0.0",
-    "nx": "17.2.8",
-    "postcss-preset-env": "9.3.0",
-    "postcss-url": "10.1.3",
-    "prettier": "3.1.0",
-    "prettier-plugin-organize-imports": "3.2.4",
-    "semver": "7.6.0",
-    "standard-version": "9.5.0",
-    "through2": "4.0.2",
-    "ts-jest": "29.1.0",
-    "ts-node": "10.9.2",
-    "tsconfig-paths": "4.1.0",
-    "typedoc": "0.25.3",
-    "typedoc-plugin-merge-modules": "5.1.0",
-    "typescript": "5.2.2",
-    "wait-on": "6.0.1",
-    "wdio-axe": "2.2.3",
-    "wdio-chromedriver-service": "6.0.4",
-    "webdriverio": "7.27.0"
-  },
-  "lint-staged": {
-    "*": [
-      "yarn nx format:write --uncommitted"
-    ],
-    "*.{js,ts,html,md,json,yml}": [
-      "yarn eslint --fix"
-    ],
-    "*.properties": [
-      "yarn nx run i18n:transform-translations"
-    ]
-  }
+    "name": "fundamental-ngx",
+    "private": true,
+    "license": "Apache-2.0",
+    "main": "dist/libs/core/index.js",
+    "description": "Fundamental Library for Angular is a themeable Fiori Angular component library.",
+    "homepage": "https://sap.github.io/fundamental-ngx",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/fundamental-ngx"
+    },
+    "engines": {
+        "yarn": ">= 1.22.19",
+        "node": ">= 18"
+    },
+    "packageManager": "yarn@1.22.18",
+    "scripts": {
+        "preinstall": "npx only-allow yarn",
+        "build": "echo Building default app && nx compile",
+        "build-docs-netlify": "yarn run build --configuration=production-unoptimized",
+        "build:umbrella": "echo Building umbrella libraries && nx run-many --target=build --all",
+        "build:umbrella:prod": "echo Building umbrella libraries PROD && yarn run build:umbrella -- --configuration=production",
+        "build:prepare": "echo Building libraries and preparing for publishing && nx run-many --target=prepare --all --pack --configuration=production --skip-nx-cache",
+        "test": "echo Running unit tests && nx run-many --target=test --all --parallel=10",
+        "app:e2e:affected": "echo Running affected e2e tests on local && nx run docs:e2e-app:affected-local",
+        "app:e2e:all": "echo Running all e2e tests on local && nx run docs:e2e-app:all-local",
+        "ng": "ng",
+        "hotfix-release": "node scripts/release-hotfix.js",
+        "start": "nx serve",
+        "test:coveralls": "cat ./coverage/libs/core/lcov.info  | ./node_modules/coveralls/bin/coveralls.js || echo -e \"Coveralls failed.\"",
+        "workspace-schematic": "./node_modules/.bin/nx workspace-schematic",
+        "nx": "nx",
+        "postinstall": "node ./decorate-angular-cli.js",
+        "prepare": "husky install",
+        "pre-commit": "lint-staged"
+    },
+    "publishConfig": {
+        "registry": "https://repository.hybris.com/api/npm/npm-repository/"
+    },
+    "dependencies": {
+        "@angular/animations": "17.3.0",
```

Comment:
would that be a problem for applications staying on version 17.0.x, 17.1.x, 17.2.x?

Reply:
Hm, I think I have to test it out

Reply:
Checked it, although at the moment it will work without any issues, when we will be using new query signals or other features that are coming with 17.3, users who use 17.0-17.2 will not be able to build their applications